flutter_local_notifications/lib/src/platform_flutter_local_notifications.dart

```diff
@@ -190,6 +190,18 @@ class IOSFlutterLocalNotificationsPlugin
         'initialize', initializationSettings.toMap());
   }
 
+  Future<bool> requestPermissions({
```

Comment:
Will need to add API docs. I can add some in directly to this PR if you're ok with that

Reply:
Will add basic docs, you can improve them if you feel like it.